bin/crypto_conditions_test_gen/conditions.py

```diff
@@ -67,6 +84,9 @@
     ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
     OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 */
+
+// THIS FILE WAS AUTOMATICALLY GENERATED -- DO NOT EDIT
+
```

Comment:
This comment is not in the generated test files. I assume that means you need to regenerate them. (See also my earlier comment about scripting the generation.)